foreman/Dockerfile

```diff
@@ -13,6 +13,12 @@ COPY data_models/dist/data-refinery-models-* data_models/
 RUN pip install \
```

Comment:
Ok, that's fine then. Maybe put in a comment in the first `pip install` statement that says that all pip installs in this file will use pip3.